src/study.js

```diff
@@ -0,0 +1,66 @@
+async function init() {
+  const elements = [...document.querySelectorAll("[data-i18n-message]")];
+  for (const element of elements) {
+    element.innerHTML = DOMPurify.sanitize(browser.i18n.getMessage(element.dataset.i18nMessage), {
+      ALLOWED_TAGS: ["a"],
+      ALLOWED_ATTR: ["href"],
+    });
+  }
+
+  const brandStrings = await browser.experiments.settings.getStrings(["brandShortName", "brandFullName"]);
+
+  function createNodes(el, tagConfig) {
+    for (const childEl of [...tagConfig.childNodes]) {
+      if (childEl.nodeName === "#text") {
+        // Ignore whitespace nodes
+        if (!/[^\W]/.test(childEl.textContent)) {
+          continue;
+        }
+        const textOutput = childEl.textContent;
+        el.appendChild(document.createTextNode(textOutput));
```

Comment:
nit: you could use `el.append(textOutput);`

Reply:
:heavy_check_mark: